### ISSUE TITLE
TypeError: Cannot assign to read only property

### ISSUE THREAD

**Issue by mariogillazaro** [szchenghuang/react-animated-ellipsis#2]
I've tried to use this component but get this:
`TypeError: Cannot assign to read only property 'fontSize' of object '#<Object>'`

This happens even if I don't pass any prop to the component.

Using it with `import ReactAnimatedEllipsis from 'react-animated-ellipsis';`

**Comment by szchenghuang** [comment]
That should make it. I don't see trouble in it.

Mind to put together a repo? I am wondering if it is related to the environment.

**Comment by szchenghuang** [comment]
Updated this package with the latest React and rewrote it to a function component with some little optimization altogether just now. It includes a [fix](https://github.com/szchenghuang/react-animated-ellipsis/commit/1732f16da381df85b9a8a71a6388d4a9ea9afff2#diff-1fdf421c05c1140f6d71444ea2b27638R22) to this issue, before I sensed the proposed PR. The issue should've been resolved now. Sorry for this awfully late cure.
